examples/keyvalue/gen-go/keyvalue/tchan-keyvalue.go

```diff
@@ -94,6 +96,41 @@ func (s *tchanAdminServer) Methods() []string {
 	}
 }
 
+// RegisterInterceptors registers the provided interceptors with the server.
+func (s *tchanAdminServer) RegisterInterceptors(interceptors ...thrift.Interceptor) {
+	if s.interceptors == nil {
+		interceptorsLength := len(interceptors)
+		s.interceptors = make([]thrift.Interceptor, interceptorsLength, interceptorsLength)
+	}
+
+	s.interceptors = append(s.interceptors, interceptors...)
```

Comment:
I don't think this works correctly.

If `s.interceptors` is `nil`, and the user tries to add 3 interceptors, you create a slice with length = 3 and capacity = 3. Then you append to that list, you're so the length now becomes 6. You have created 3 `nil` entries.